src/nemo-connectivity/mobiledataconnection.cpp

```diff
@@ -273,7 +321,11 @@ void MobileDataConnectionPrivate::updateDataContext()
         }
         connectionContext->setContextPath(inetContextPath);
     } else if (hasDataContext() && !connectionManager->powered()) {
+        qCDebug(CONNECTIVITY, "######## Set powered ON");
         connectionManager->setPowered(true);
+
+        if (networkTechnology)
```

Comment:
I believe our C++ coding style says always use curlies, but then again the usage inside libconnman-qt seems mixed, so feel free to ignore.